src/pages/BlockInfoPage.tsx

```diff
@@ -209,28 +207,21 @@ const TransactionRow: FC<TransactionRowProps> = ({ transaction }) => {
   return (
     <>
       <TableRow key={t.hash} isActive={detailOpen} onClick={toggleDetail}>
-        <td>
-          <TransactionIcon />
-        </td>
-        <td>
-          <TightLink to={`/transactions/${t.hash}`} text={t.hash} maxWidth="150px" />
-        </td>
-        <td>
+        <TransactionIcon />
+        <TightLink to={`/transactions/${t.hash}`} text={t.hash} maxWidth="150px" />
+        <span>
           {t.inputs ? t.inputs.length : 0} {t.inputs && t.inputs.length === 1 ? 'address' : 'addresses'}
-        </td>
-        <td>
-          <ArrowRight size={15} />
-        </td>
-        <td>
+        </span>
+        <ArrowRight size={15} />
+        <span>
           {outputs ? outputs.length : 0} {outputs?.length === 1 ? 'address' : 'addresses'}
-        </td>
-        <td>
-          <Badge
-            type="neutralHighlight"
-            amount={outputs?.reduce<bigint>((acc, o) => acc + BigInt(o.attoAlphAmount), BigInt(0))}
-            floatRight
-          />
-        </td>
+        </span>
+        <Badge
+          type="neutralHighlight"
+          amount={outputs?.reduce<bigint>((acc, o) => acc + BigInt(o.attoAlphAmount), BigInt(0))}
```

Comment:
nit: I know this line was not created in this PR, but... 😅 I think the render function would look much nicer if this reducer was moved into a `totalAmount` variable.

Reply:
Done